app/src/main/res/layout/fragment_event_details.xml

```diff
@@ -152,11 +152,12 @@
             android:minHeight="10dip"
             android:paddingLeft="12dp"
             android:paddingRight="12dp"
+            android:paddingBottom="12dp"
             android:progress="0" />
 
     </LinearLayout>
 
-    <Button
+    <!--Button
```

Comment:
that button is not required now. Since user can navigate to attendees page directly using navigation view.